ksqldb-common/src/main/java/io/confluent/ksql/util/BytesUtils.java

```diff
@@ -110,6 +110,42 @@ public static String encode(final byte[] value, final String encoding) {
     return Arrays.copyOfRange(getByteArray(buffer), start, end);
   }
 
+  public static int indexOf(final byte[] array, final byte[] target, final int fromIndex) {
+    for (int i = fromIndex; i < array.length; i++) {
+      if (array[i] == target[0]) {
+        if (arrayEquals(array, i, target, 0, target.length)) {
+          return i;
+        }
+      }
+    }
+
+    return -1;
+  }
+
+  @SuppressWarnings("ParameterName")
```

Comment:
I had to add this line because spotbugs was complaining that `aFromIndex` and `bFromIndex` weren't matching the parameter names. I changed them to a shorter name, but it was still complaining.